nixos/modules/services/monitoring/vnstat.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.vnstat;
+in {
+  options.services.vnstat = {
+    enable = mkEnableOption "If enabled, vnstatd will be started to update the network usage statistics.";
```

Comment:
This will render poorly. `mkEnableOption` already adds some boilerplate text to the description. Also, I think the consensus is to phase out usage of `mkEnableOption`.